projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration.ts

```diff
@@ -306,6 +324,21 @@ export function checkAttributeDisplayed(
   cy.get(`#${attributeId}`).should('be.visible');
 }
 
+/**
+ * Verifies if all passed bundle attributes are displayed
+ *
+ * @param {string[]} attributeHeaders
+ */
+export function checkBundleAttributeDisplayed(
```

Comment:
Please, remane this method to **checkAttributeHeaderDisplayed** because the **cx-configurator-attribute-header** component is not bundel specific